L1Trigger/L1THGCal/test/HGCalTriggerGeomTesterV9Imp3.cc

```diff
@@ -1014,6 +1023,21 @@ void HGCalTriggerGeomTesterV9Imp3::fillTriggerGeometry()
       triggerCellWaferV_ = id_si_trig.waferV();
       triggerCellU_ = id_si_trig.triggerCellU();
       triggerCellV_ = id_si_trig.triggerCellV();
+
+      if (triggercell_cells.second.begin() == triggercell_cells.second.end()) {
+        throw cms::Exception("BadGeometry") << "HGCalTriggerGeometry: No cells!";
+      }
+      const HGCSiliconDetId& firstCellId(*triggercell_cells.second.begin());
+      if (firstCellId.det() == DetId::HGCalEE) {
+        wafertype = triggerGeometry_->eeTopology().dddConstants().waferType(firstCellId, true);
+      } else if (firstCellId.det() == DetId::HGCalHSi) {
+        wafertype = triggerGeometry_->hsiTopology().dddConstants().waferType(firstCellId, true);
```

Comment:
No reason, as far as I can see. Despite not finding the default value (it is not specified [here](https://github.com/hgc-tpg/cmssw/blob/729497c9794c3411c4f5e9a7f9f5cdbf18753123/Geometry/HGCalCommonData/src/HGCalDDDConstants.cc#L1538)), [this](https://github.com/hgc-tpg/cmssw/blob/729497c9794c3411c4f5e9a7f9f5cdbf18753123/Geometry/HGCalCommonData/src/HGCalDDDConstants.cc#L1558) is way more readable than [this](https://github.com/hgc-tpg/cmssw/blob/729497c9794c3411c4f5e9a7f9f5cdbf18753123/Geometry/HGCalCommonData/src/HGCalDDDConstants.cc#L1542-L1556). I've rerun the code with the modification, and no issues were observed.

Reply:
The default value is specified in the header [here](https://github.com/hgc-tpg/cmssw/blob/ee68b23b3b289651c03e712df66ed9a30798709e/Geometry/HGCalCommonData/interface/HGCalDDDConstants.h#L210)

Reply:
Since there is a default value, I think it would be better to just drop the `false` (i.e. `waferType(firstCellId)`) as this looks more like an "internal" argument and I guess the user don't need to know about it.

Reply:
Thanks. The new version is using ```fromFile=false``` explicitly for clarity, but I can remove it if you prefer.

Reply:
Yes, I think it would be better to remove it, as in my opinion this is not something that should be exposed on the user side, and if at some point the default is changed to `true` it is better to have it automatically propagated